docs/api-reference/next/image.md

```diff
@@ -43,17 +47,94 @@ function Home() {
-- `layout` - The rendered layout of the image. If `fixed`, the image dimensions will not change as the viewport changes (no responsiveness). If `intrinsic`, the image will scale the dimensions down for smaller viewports but maintain the original dimensions for larger viewports. If `responsive`, the image will scale the dimensions down for smaller viewports and scale up for larger viewports. If `fill`, the image will stretch both width and height to the dimensions of the parent element. Default `intrinsic`.
-- `sizes` - Defines what proportion of the screen you expect the image to take up. Recommended, as it helps serve the correct sized image to each device. [More info](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-sizes).
-- `quality` - The quality of the optimized image, an integer between 1 and 100 where 100 is the best quality. Default 75.
-- `loading` - The loading behavior. When `lazy`, defer loading the image until it reaches a calculated distance from the viewport. When `eager`, load the image immediately. Default `lazy`. [More info](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-loading)
-- `priority` - When true, the image will be considered high priority and [preload](https://web.dev/preload-responsive-images/). Should only be used when the image is visible above the fold.
-- `unoptimized` - When true, the source image will be served as-is instead of changing quality, size, or format.
-- `unsized` - **Deprecated** When true, the `width` and `height` requirement can by bypassed. Use the `layout` property instead!
+Recommended, as it helps serve the correct sized image to each device.
+
+[Learn more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-sizes).
+
+### quality
+
+The quality of the optimized image, an integer between 1 and 100 where 100 is the best quality.
+
+Default 75.
+
+### loading
+
+The loading behavior of the image.
+
+When `lazy`, defer loading the image until it reaches a calculated distance from the viewport.
+
+When `eager`, load the image immediately.
+
+Default `lazy`.
+
+[Learn more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-loading)
+
+### priority
+
+When true, the image will be considered high priority and [preload](https://web.dev/preload-responsive-images/).
+
+Should only be used when the image is visible above the fold.
+
+### unoptimized
+
+When true, the source image will be served as-is instead of changing quality, size, or format.
+
+### unsized
+
+**Deprecated** - Use the [layout](#layout) property instead.
+
+When true, the `width` and `height` requirement can by bypassed.
+
+### Other Properties
 
 All other properties on the `<Image>` component will be passed to the underlying `<img>` element.
+
+## Related
+
+We recommend you to read [Image Optimization](/docs/basic-features/image-optimization.md).
```

Comment:
Let's make this work like in the other sections, e.g:

![image](https://user-images.githubusercontent.com/4278345/98056115-158e7d80-1e0d-11eb-82d0-95fded6916a5.png)

Reply:
Sounds good, I updated the Related section